packages/new-editor/src/components/editor/variables/detail/Variable.tsx

```diff
@@ -0,0 +1,39 @@
+import { Fieldset, Input, SimpleSelect, Textarea, ToolbarTitle } from '@axonivy/ui-components';
+import { Metadata, type Variable } from '../../data/Variable';
+
+type VariableProps = {
+  variable: Variable;
+};
+
+export const VariableDetail = ({ variable }: VariableProps) => {
+  const metadataOptions = Object.keys(Metadata)
+    .filter(key => isNaN(Number(key)))
+    .map(key => {
+      const option = key.toString();
+      return { value: option, label: option };
+    });
+  metadataOptions.unshift({ value: 'None', label: 'None' });
+  const selectedMetadataOption = variable.metadata === undefined ? 'None' : Metadata[variable.metadata];
```

Comment:
I don't like this enum handling stuff and having to introduce an extra option to not select anything. Any ideas on improvements?

Reply:
yes enums technially do not exist in ts. or better they really do not exist in js. ts does some "magic" here to fake them in js...

I would suppose something like in the inscription-view:
```ts
export const RESPONSIBLE_TYPE = {
  ROLE: 'Role',
  ROLE_FROM_ATTRIBUTE: 'Role from Attr.',
  USER_FROM_ATTRIBUTE: 'User from Attr.',
  DELETE_TASK: 'Nobody & delete'
} as const satisfies Record<WfActivatorType, string>;

Object.entries(RESPONSIBLE_TYPE)
        .filter(([value]) => !(props.optionFilter && props.optionFilter.includes(value as WfActivatorType)))
        .map(([value, label]) => ({ label, value }));
```

or in the form-editor:
```ts
const typeOptions: FieldOption<InputType>[] = [
  { label: 'Email', value: 'EMAIL' },
  { label: 'Password', value: 'PASSWORD' },
  { label: 'Text', value: 'TEXT' }
] as const;
```

at the moment you only "fake" the metas, so totally fine to hold this type as simple as possible.